light_dark_mode/src/components/Navbar.js

```diff
@@ -0,0 +1,28 @@
+import React from "react"
+
+export default function Navbar(props) {
+    return (
+         <nav 
+            className={props.darkMode ? "dark": ""}
+        >
+            <img 
+                className="nav--logo_icon"
+                src="./images/react-icon-small.png"
+            />
+            <h3 className="nav--logo_text">ReactFacts</h3>
+            
+            <div 
+                className="toggler" 
+            >
+                <p className="toggler--light">Light</p>
+                <div 
+                    className="toggler--slider"
+                    onClick={props.toggleDarkMode}
+                >
+                    <div className="toggler--slider--circle"></div>
```

Comment:
please remove `</div>` and change to `<div className="toggler--slider--circle" />`